src/workflows/orders.ts

```diff
@@ -7,7 +7,19 @@ import {
   OrdersApiCreateOrderRequest,
 } from '../api';
 import { serializeSignature, sign, signRaw } from '../utils';
+import { WalletConnection } from '../types/index';
 
+type CreateOrderWorkflowWithSignerRequest = WalletConnection & {
```

Comment:
shouldn't these types be in a `types` folder ?

Reply:
it is a private type that only works for those function inputs.